### PR TITLE
Add InstanceID deprecation warning

### PR DESCRIPTION
**Pull request by chliangGoogle**
Adding deprecation warnings that are documented in the token API review doc.

### CODE REVIEW

File: Firebase/InstanceID/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 2020-09 -- 4.7.0
+- [changed] Added deprecation warning for InstanceID. InstanceID is deprecated, for app instance identity handling, use FirebaseInstallations. For FCM registration token handling, use FirebaseMessaging. (#6585)
```
Comment (paulb777) [inline Firebase/InstanceID/CHANGELOG.md:2]: Period after "InstanceID is deprecated."

File: FirebaseMessaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 2020-09 -- v.4.7.1
+- [added] InstanceID is deprecated, add macro to supress deprecation warning. (#6585)
```
Comment (paulb777) [inline FirebaseMessaging/CHANGELOG.md:2]: suppress

**Review by paulb777 — COMMENTED** [review]
Fix CI

**Review by maksymmalyhin — APPROVED** [review]
LGTM on CI green

**Review by paulb777 — APPROVED** [review]

**Pull request merged**